debian/postinst

```diff
@@ -22,6 +22,31 @@ case $1 in
 	systemctl enable systemd-networkd.service
 
 	systemctl disable walinuxagent.service
+
+	if ! id -u postgres > /dev/null; then
```

Comment:
can you also comment on why we do this configuration here, rather then some place else (e.g. the delphix-platform service)? I we have to do it here for a good reason, so it'd be good to explain that reason, so it's not accidentally moved without realizing the implications.